### PR TITLE
Updating grafana image.

### PR DESCRIPTION
**Pull request by vishh**
cc @thucatebay

### CODE REVIEW

File: deploy/kube-config/influxdb/influxdb-grafana-controller.json [hunk 1]
```diff
@@ -24,30 +24,37 @@
...
+			"name": "influxdb-store",
+			"emptyDir": {}
 		    }
+
 		],
 		"containers": [
 		    {
 			"image": "kubernetes/heapster_influxdb:v0.4",
 			"name": "influxdb",
-			"ports": [
```
Comment (jimmidyson) [inline deploy/kube-config/influxdb/influxdb-grafana-controller.json]: Why are these ports removed?

Comment (jimmidyson) [inline deploy/kube-config/influxdb/influxdb-grafana-controller.json]: Ah they're in same pod.

Comment (vishh) [inline deploy/kube-config/influxdb/influxdb-grafana-controller.json]: Yeah

**Comment by vishh** [comment]
@cadvisorJenkinsBot: test this please

**Pull request merged**